src/foam/nanos/cron/SimpleIntervalScheduleView.js

```diff
@@ -27,6 +27,9 @@ foam.CLASS({
     ^halfWidth {
       grid-column: span 6;
     }
+    ^halfWidth.foam-u2-PropertyBorder {
+      justify-content: flex-start;
```

Comment:
Why is this required?

Reply:
@sarthak-marwaha because I tried to align these two fields.
<img width="735" alt="Screen Shot 2022-10-05 at 12 17 34 PM" src="https://user-images.githubusercontent.com/55885868/194112685-9eff8b19-3bfa-43e1-a8ae-1ef006155ed0.png">